spec/models/upload_spec.rb

```diff
@@ -434,41 +434,6 @@ def enable_secure_media
     end
   end
 
-  context ".signed_url_from_secure_media_url" do
```

Comment:
It's not clear to me why these specs have been removed.

Reply:
That makes 2 of us! Looks like it was just some merge shenanigans from those other secure media fixes I made, fixed now.